_posts/en/newsletters/2022-06-22-newsletter.md

```diff
@@ -0,0 +1,230 @@
+clients and services, announcements of new releases and release
+candidates, and summaries of notable changes to popular Bitcoin
+infrastructure software.
+
+## News
+
+- **Full replace by fee:** [two][bitcoin core #25353] pull
+  [requests][bitcoin core #25373] have been opened to add support to
+  Bitcoin Core for full Replace By Fee ([RBF][topic rbf]) as an option
+  that is off by default.  If enabled, any unconfirmed transaction in
+  that node's mempool could be replaced by an alternative version of
+  that transaction which pays a higher feerate (among other rules).
+
+  Currently Bitcoin Core only allows RBF if the version of the
+  transaction to be replaced has a signaling bit enabled, as defined in
+  [BIP125][].  This creates a challenge for multiparty contract
+  protocols, such as LN and [DLCs][topic dlc], where it is sometimes
+  possible for one party to remove the BIP125 signal from a transaction
+  in order to prevent other parties from using transaction replacement.
+  This can lead to delays, and in the worst case it may lead to a loss
+  of funds in protocols that depend on timely confirmation (such as for
+  [HTLCs][topic htlc]).
+
+  [One of the PRs][bitcoin core #25353] quickly received significant
+  developer support.  Because it only adds the ability to enable full
+  RBF---but does not enable it by default---it does not change Bitcoin
+  Core's current default behavior.  In the long-term, some developers
+  will likely advocate for enabling full RBF by default, so a thread was
+  [started][rbf discussion] on the Bitcoin-Dev mailing list this week to
+  give developers of services, applications, and alternative full node
+  software a chance to argue against the direction of providing a full
+  RBF option and perhaps eventually enabling it by default.
+
+- **Hertzbleed:** a recently [disclosed][hertzbleed] security
+  vulnerability affecting many (perhaps all) popular laptop, desktop,
+  and server CPU processors may allow attackers to discover private keys
+  when those keys are being used to create signatures for Bitcoin
+  transactions (or perform other similar operations).  The noteworthy
+  aspect of this attack is that it may affect signature generation code
+  that was specifically written to always use the same type and number
+  of CPU operations in order to prevent leaking information to
+  attackers.
+
+  Exploiting the vulnerability would require an attacker to measure
+  either the power consumption of a CPU chip or measure the duration
+  of parts of the signing operation.  Ideally for an attacker, they
+  would be able to take measurements while a user creates many
+  signatures using the same private key.  As such, the vulnerability is
+  more likely to affect frequently used hot wallets, such as those used
+  by hosted services and LN routing nodes.  Mostly or entirely offline
+  wallets that are used in secure environments would be much more
+  resistant to attacks.
```

Comment:
Would I understand correctly that address reuse exacerbates this? Perhaps we could call that out specifically to add to the well-known reasons for why address reuse is a bad practice.